pycm/generate_random_data.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""This file contains a function to generate a random confusion matrix."""
+import numpy as np
+from enum import Enum
+
+
+class ClassDistributionScenario(Enum):
+    """
+    Enum to represent different scenarios for generating class percentages.
+
+    - UNIFORM: All classes have equal representation.
+    - MAJORITY_CLASS: Only one class has a majority representation, others are minority.
+    - MINORITY_CLASS: Only one class has a minority representation, others are majority.
```

Comment:
Again, I think `others are minority/majority` should be true.
It can be misleading, for example, when the rates are like [10%, 30%, 30%, 30%], we can only say the first class is a minority. I don't consider each of the other classes separately to be the majority.